changelog_unreleased/api/12740.md

```diff
@@ -1,4 +1,4 @@
-#### [BREAKING] standalone version export style changed (#12740 by @fisker)
+#### [Breaking] standalone version export style changed (#12740 by @fisker)
 
 The `esm/standalone.mjs` entry has changed export style from default export to named export.
```

Comment:
That was fast! :)
My only comment is that there is no need to revert this file.